samples/prerequisites.md

```diff
@@ -1,23 +1,51 @@
-If you're new to .NET make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
+If you're new to .NET, make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
 If you're already familiar with .NET make sure you've [downloaded and installed the .NET 5 SDK](https://www.microsoft.com/net/download/core).
 
-The following pre-requisites need to be installed for building .NET 5 projects with CoreRT:
+The following pre-requisites need to be installed for building .NET 5 projects with CoreRT.
 
 # Windows
 
-* Install [Visual Studio 2019](https://visualstudio.microsoft.com/vs/community/), including Visual C++ support.
+## Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) with Visual C++ Build Tools
+
+The free [Community](https://visualstudio.microsoft.com/vs/community/) edition or higher is recommended.
+If you wish to save disk space and do not need Visual Studio IDE, here is the minimal [Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019)
```

Comment:
These steps feel a lot more complicated compared to launch Visual Studio Installer and check "C++ suport". I think we should keep this simple option as the preferred option. We can have these disk-space saving steps as "## Advanced Alternative: Minimum Visual C++ Build Tools installation" or something.

Reply:
Makes sense.